v3/lints/cabf_ev/lint_ev_organization_id_missing_test.go

```diff
@@ -0,0 +1,39 @@
+package cabf_ev
+
+/*
+ * ZLint Copyright 2020 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+var tests = map[string]lint.LintStatus{
+	"EvAltRegNumCert52NoOrgId.pem":                       lint.NA,
+	"EvAltRegNumCert55OrgIdExtMissing.pem":               lint.NE,
+	"EvAltRegNumCert67ValidNtrWithOrgIdExt.pem":          lint.NE,
+	"EvAltRegNumCert55OrgIdExtMissingNewerDate.pem":      lint.Error,
+	"EvAltRegNumCert67ValidNtrWithOrgIdExtNewerDate.pem": lint.Pass,
```

Comment:
Do the numbers in the filenames have meaning here? Should they?

I don't have strong feelings, but it seems that `EvAltregNumCertXX` is... less than descriptive. And the non-sequentially of these suggests that any implied semantics (e.g. 52 < 55) would be defeated if 53/54 are introduced.

Reply:
> Do the numbers in the filenames have meaning here? Should they?

I admit that I wondered the exact same thing as I simply took the test data from #399 and spot checked them, but otherwise forgot about the filenames. I'll try to figure out if there are any semantics attached there, otherwise I'll just rename these.